README.md

```diff
@@ -127,6 +123,33 @@ If you're not running a screen yet, type `screen` to start one.
 Enjoy your fishy friend!
 
 
+## Adding Screenguppy to Tmux
+
+Open your `~/.tmux.conf` file in a text editor, or create it if it's not
+already there. Look for any lines setting either `status-left` or
+`status-right`. If none are present, just add the following line:
+
+```
+set -g status-left '#(screenguppy)'
+```
+
+If you do have status lines in the config, and add to one of them the text
+`#(screenguppy)`. You can limit the width that the fish will swim in
+by adding a number afterwards: `#(screenguppy 20)`.
```

Comment:
tmux defaults to a width of 10 for status-left, this would need to use the same value or have a blurb about also adding `set -g status-left-width N` to a matching value.

Reply:
Also worth adding `set -g status-interval 1` as the default of 15 seconds means the display isn't refreshed fast enough to see the fish moving smoothly (it's only visible every two minutes or so with the default width and refresh interval).